README.md

```diff
@@ -181,5 +181,22 @@ For detailed instructions, see [building](doc/BUILDING.md).
 - See the [contribution guide](doc/contributing/PROCESS.md)
 - Join our [working group](https://github.com/OpenAssetIO/OpenAssetIO-WG) meetings
 
+> **Warning**
+> OpenAssetIO is a project used in commercial workflows, and due to this
+> need to work stably and reliably, it is a project with rigid
+> standards.
+>
+> Combined with its inherently abstract nature, it is not a good project
+> to dip your toes lightly into. Beyond minor documentation tweaks, it
+> is difficult to find work appropriate for less experienced
+> implementors.
```

Comment:
[fix] Be good to keep it away from directly negative phrasing, how about:
```suggestion
> Combined with its inherently abstract nature, it is a difficult project
> for casual contribution. We are a small team, and welcome anyone
> who can directly help contribute to the core tasks outlined in our 
> [roadmap](ROADMAP.md). We are however, at present, unable to
> provide "good first tasks" or a gentle, guided, on-ramp for those new 
> to the problem space.
```

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1188/commits/81fd7898743250ff2645bd577f7a3a8af743b644